### ISSUE TITLE
Option to disable automatic engagement of the approach-phase (workaround) 

### ISSUE THREAD

**Issue by oMrSmith** [flybywiresim/a32nx#6176]
### Aircraft Version

Stable

### Description

Today I discovered your mod and I had my first enjoyable 
airliner-flight in FS2020 ever. Im coming from XP...
You have done an impressive work!

I noticed only one issue worth mentioning:
I was flying into LGAV ILS 03L Z
and the approach phase engaged way too early.

It seems like the FMC doesn't take the current ALT
into account. I was already lower than planed,
cause ATC advised me to descend down to
the platform altitude already...

I was wondering if a possible work around could be
to optionally disable the automatic switch over to the approach phase...

**Comment by Benjozork** [comment]
DECEL will engage approach phase if you are below 9500ft, in NAV, LOC or LOC*.

Being in HDG is one fix that would inhibit auto engagement.

**Comment by oMrSmith** [comment]
Ah, thanks for your advise! Could you perhaps help me on another issue?
Is there any way to bind a keyboard key to (properly) in- and decrease thrust?
Unfortunately FS2020 does not properly support my Joystick (Thrustmaster T.Flight X).

**Comment by oMrSmith** [comment]
I flew the route again. Made absolutely sense, that the APPR PHASE 
triggered as I passed the DECEL point.

But should the DECEL point not _move back_
if my current altitude is lower than written in the flight plan?

![image](https://user-images.githubusercontent.com/78870770/141681325-99be7d69-b776-45c0-a0b0-d49284590ef0.png)

I was 33nm away from the airport, the approach segment of the route had not even begun
and the APPR PHASE triggered even before I passed the DECEL point on the map.
